framework/README.md

```diff
@@ -12,8 +12,9 @@ All modules can be easily replaced by other libraries when needed.
 
 ## Installation
 
+Set the target URL to the desired tarball distribution permalink.
 ```
-npm install lisk-service-framework
+npm install https://github.com/LiskHQ/lisk-service/raw/2fbc65f49ae5c5aaa3689667051dbd4082b8c25a/framework/dist/lisk-service-framework-1.4.10.tgz
```

Comment:
is this how we currently doing it?

Reply:
Yes!

Reply:
Probably every-time we update the version we have remember to update this version i guess. 
we need to find a better approach, probably if we need to publish the `lisk-service-framework` we should consider doing that in this release?

Reply:
Everytime we make changes to the framework, we package the distribution and check it in the repo itself under the path: `./framework/dist`